frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableColumnList/TableColumn/TableColumn.tsx

```diff
@@ -77,16 +72,12 @@ export const TableColumn: FC<TableColumnProps> = ({
           <Cardinality
             direction="right"
             cardinality="ONE_TO_ONE"
-            isHighlighted={
-              isHighlighted || highlightedHandles.includes(handleId)
-            }
+            isHighlighted={isHighlighted}
```

Comment:
📝 The loop in `highlightedHandles.includes(handleId)` was being executed multiple times, so it was summarised.